config.json

```diff
@@ -251,6 +251,12 @@
       ]
     },
     {
+      "slug": "zebra-puzzle",
+      "difficulty": 1,
+      "topics": [
+      ]
+    },
+    {
```

Comment:
Hmm, see, if we just look at `minesweeper`, it seems they are kind of similar difficulty, but I think `minesweeper` could stand to get moved to around where `change` is. This current placement is about 2/3 of the way through the track so I don't think we'll surprise anyone too badly, and we can look at the reordering later.